Daniel-L-Ross/kennels
# Dr employee directory

# Description

Refactored employee page to display a simplified list. 
Created a dynamic detail page to display specific employee information.
Create a button to delete an employee from the database.

Fixes #25 
Fixes #28 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Clicking an employee name should direct to a page displaying info about that employee.
Clicking the "Remove Employee" button on the employee's detail page should delete that employee and redirect to the list of employees.


# Checklist:
